numaprom/__init__.py

```diff
@@ -21,3 +23,6 @@ def get_logger(name):
     pl_logger.setLevel(logging.ERROR)
     pl_logger.addHandler(stream_handler)
     return logger
+
+
+__all__ = ["UnifiedConf", "MetricConf", "ServiceConf", "NumapromConf", "get_logger"]
```

Comment:
Are we importing this `get_logger` anywhere?

Reply:
yes, everywhere

Reply:
won't this `logging.getLogger(__name__)` do?

Reply:
for setting pytorch_lightning logs we had to make it a method call it explicitly